code/parachain/runtime/picasso/src/lib.rs

```diff
@@ -641,7 +641,7 @@ parameter_types! {
 	pub const CrowdloanRewardsLockId: LockIdentifier = *b"clr_lock";
 	pub const InitialPayment: Perbill = Perbill::from_percent(50);
 	pub const OverFundedThreshold: Perbill = Perbill::from_percent(1);
-	pub const VestingStep: Moment = (7 * DAYS as Moment) * (MILLISECS_PER_BLOCK as Moment);
+	pub const VestingStep: Moment = (DAYS as Moment) * (MILLISECS_PER_BLOCK as Moment);
```

Comment:
Need to do another PR for this, so that I can get it in the release faster (Let me do it)

Reply:
👍